shared-test-setup/src/main/java/com/exasol/projectkeeper/test/NpmProjectFixture.java

```diff
@@ -0,0 +1,73 @@
+package com.exasol.projectkeeper.test;
+
+import static java.util.Collections.emptySet;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Locale;
+
+import com.exasol.projectkeeper.shared.config.*;
+import com.exasol.projectkeeper.shared.config.ProjectKeeperConfig.Builder;
+
+public class NpmProjectFixture extends BaseProjectFixture {
+    private static final String PACKAGE_FILE_NAME = "package.json";
+    private static final String PROJECT_VERSION = "1.2.3";
+
+    public NpmProjectFixture(final Path projectDir) {
+        super(projectDir);
+    }
+
+    public ProjectKeeperConfig.Builder createDefaultConfig() {
+        return ProjectKeeperConfig.builder().sources(List.of(
+                Source.builder().modules(emptySet()).type(SourceType.NPM).path(Path.of(PACKAGE_FILE_NAME)).build()));
+    }
+
+    public void prepareProjectFiles(final Builder configBuilder) {
+        this.writeConfig(configBuilder);
+        this.prepareProjectFiles();
+        execute(projectDir, "npm" + suffixForWindows(".cmd"), "install");
+    }
+
+    private String suffixForWindows(final String suffix) {
```

Comment:
Any specific reason you're passing in `.cmd` ? Why not make the function parameterless and let it return  `.cmd`?

Reply:
Hmmm.... good point ;)

Reply:
Merged this PR too early. I fixed this review finding in another PR: https://github.com/exasol/project-keeper/pull/629